js/chatbot/shared/ChatBot.svelte

```diff
@@ -306,7 +306,7 @@
 		background: var(--background-fill-secondary);
 		width: calc(100% - var(--spacing-xxl));
 		color: var(--body-text-color);
-		font-size: var(--text-lg);
+		font-size: var(--body-text-size);
```

Comment:
I think this will cause a breaking change for demos that use the default theme without specifying a text size, because the default "body-text-size" is medium.

Reply:
Since the previous behaviour was broken, I'd be inclined to call this a bugfix.

Reply:
Nice! LGTM

> My concern with this fix is that in a demo (with a theme) with a chatbot and multiple other components, the body text sizes won't be consistent throughout each component. Some custom CSS could be used to fix this but I'm wondering whether this might cause developer frustration as well.

That's true, although that is also the case right now with the default theme (chatbot size is larger than main body text) and we haven't heard any issues with it.